app/controllers/merge_requests_controller.rb

```diff
@@ -61,6 +61,8 @@ def next_page_path
       merging_organisations_merge_request_path(@merge_request)
     when "merging_organisations"
       merge_date_merge_request_path(@merge_request)
+    when "helpdesk_ticket"
+      merge_request_path(@merge_request)
     when "merge_date"
       helpdesk_ticket_merge_request_path(@merge_request)
```

Comment:
Could we reorder these so that it's in the order that the questions are asked? 👀